### PR TITLE
KEYCLOAK-12958 Preview feature profile for WebAuthn

### PR DESCRIPTION
**Pull request by mabartos**

### CODE REVIEW

File: server_admin/topics/authentication/webauthn.adoc [hunk 1]
```diff
@@ -4,7 +4,9 @@
 
 {project_name} provides the limited support for https://www.w3.org/TR/webauthn/[W3C Web Authentication (WebAuthn)]. {project_name} works as a WebAuthn's https://www.w3.org/TR/webauthn/#rp-operations[Relying Party (RP)].
 
-IMPORTANT: Please note that WebAuthn support is still in development and not yet complete, so we recommend that you use this feature experimentally. Also, this support's specification and user interfaces may change.
```
Comment (mposolda) [inline server_admin/topics/authentication/webauthn.adoc:7]: As I mentioned to you in the private chat, we are in the end going to have WebAuthn enabled by default in the project, but marked as preview in the product. So is it possible to change the docs according to it?

We can possibly keep the previous paragraph "IMPORTANT" just for the project and for the product use the block you added with ":tech_feature_name: WebAuthn" etc?

It is possible to ask somehow in the docs whether you are rendering project and product. Hopefully you are able to figure that and let me know just if not :)

File: server_installation/topics/profiles.adoc [hunk 2]
```diff
@@ -60,6 +60,11 @@ endif::[]
 |Upload scripts through the {project_name} REST API
 |No
 |Deprecated
+
```
Comment (mposolda) [inline server_installation/topics/profiles.adoc:63]: s it possible to change this part based on whether we use project (enabled by default) or product (preview) ?

File: upgrading/topics/keycloak/changes.adoc [hunk 3]
```diff
@@ -1,5 +1,13 @@
 == Migration Changes
 
+=== Migration to 9.0.1
```
Comment (mposolda) [inline upgrading/topics/keycloak/changes.adoc]: Is it possible to remove this? We're still keeping the WebAuthn in the project enabled by default as mentioned above.

**Review by mposolda — CHANGES_REQUESTED** [review]
@mabartos Thanks for the PR. There will be some changes needed due the fact that we're enabling WebAuthn by default in the project, but leaving as "preview" in the product. See inline comments for the details.

**Review by andymunro — DISMISSED** [review]

File: server_admin/topics/authentication/webauthn.adoc (hunk 1 above)
Comment (mabartos) [reply server_admin/topics/authentication/webauthn.adoc:7]: Ok, thanks.

File: server_installation/topics/profiles.adoc [hunk 4]
```diff
@@ -60,6 +60,19 @@ endif::[]
...
+|web_authn
+|W3C Web Authentication (WebAuthn)
+
+ifeval::[{project_community}==true]
+|Yes
+|Experimental
```
Comment (mabartos) [inline server_installation/topics/profiles.adoc:69]: @mposolda I don't know, if the _Supported level_ can be `Experimental` for project?!

Comment (mposolda) [reply server_installation/topics/profiles.adoc:69]: @mabartos I think it is fine and I don't have any better idea. So i would keep it like that :)

Comment (mposolda) [reply server_installation/topics/profiles.adoc:69]: @mabartos I think it is fine and I don't have any better idea. So i would keep it like that :)

**Review by mposolda — APPROVED** [review]
@mabartos Approving, Thanks

**Pull request merged**